app/models/PensionSchemeAdministrator.scala

```diff
@@ -43,31 +45,69 @@ object NumberOfDirectorOrPartnersType {
   implicit val formats = Json.format[NumberOfDirectorOrPartnersType]
 }
 
-case class CorrespondenceCommonDetail(addressDetail: AddressType, contactDetail: ContactDetails)
+case class CorrespondenceCommonDetail(addressDetail: Address, contactDetail: ContactDetails)
 object CorrespondenceCommonDetail {
   implicit val formats = Json.format[CorrespondenceCommonDetail]
+
+  val apiReads : Reads[CorrespondenceCommonDetail] = (
+    (JsPath \ "directorContactDetails").read(ContactDetails.apiReads) and
+      (JsPath \ "directorAddress").read[Address]
+    )((contactDetails,address)=>CorrespondenceCommonDetail(address,contactDetails))
 }
 
 case class DirectorOrPartnerDetailTypeItem(sequenceId: String, entityType: String, title: Option[String] = None,
                                            firstName: String, middleName: Option[String] = None, lastName: String,
-                                           dateOfBirth: String, referenceOrNino: Option[String] = None,
+                                           dateOfBirth: DateTime, referenceOrNino: Option[String] = None,
```

Comment:
This should probably be a `LocalDate` we will also need to make sure that it is formatted in the right way.